t/tasks/inventory/generic/dmidecode/memory.t

```diff
@@ -844,6 +868,7 @@ my %tests = (
             CAPACITY         => '1024',
             CAPTION          => 'J8J1',
             MANUFACTURER     => undef,
+            MODEL            => '0x4D332037385432393533455A332D43453620',
```

Comment:
Here if you use hex2char() api on that string you'll obtain `M3 78T2953EZ3-CE6`, that's look nice and I suppose this string is a wrong encoding from the manufacturer. It is safe to use this api on part-number before setting it in inventory.